makefile

```diff
@@ -44,6 +44,9 @@ setup-istio:
 setup-addons:
 	kubectl apply -f  monitoring/istio-addons/
 
+deploy-rabbitmq-operator:
```

Comment:
I think this should go along wiht `deploy-app`, as we are essentially replacing the single pod rabbitmq deployment with this.